File: CPP/389.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
    	char total = s[0];
        for (int i = 1; i < s.length(); ++i) {
        	total ^= s[i];
        }
        for (int i = 0; i < t.length(); ++i) {
        	total ^= t[i];
        }
        return total;
    }
};

File: Java/389.java
public class Solution {
    public char findTheDifference(String s, String t) {
    	char[] ch_s = s.toCharArray();
    	char[] ch_t = t.toCharArray();
        char left = 0;
        for (char each : ch_s) {
        	left ^= each;
        }
        for (char each : ch_t) {
        	left ^= each;
        }
        return left;
    }
}